imageloading-core/src/main/java/dev/chrisbanes/accompanist/imageloading/ImageLoad.kt

```diff
@@ -72,18 +72,7 @@ fun <R : Any, TR : Any> ImageLoad(
         mutableStateOf<ImageLoadState>(ImageLoadState.Loading)
     }
 
-    // This may look a little weird, but allows the LaunchedEffect callback to always
-    // invoke the last provided [onRequestCompleted].
-    //
-    // If a composition happens *after* LaunchedEffect has launched, the given
-    // [onRequestCompleted] might have changed. If the actor lambda below directly referenced
-    // [onRequestCompleted] it would have captured access to the initial onRequestCompleted
-    // value, not the latest.
-    //
-    // This `callback` state enables the actor lambda to only capture the remembered state
-    // reference, which we can update on each composition.
-    val callback = remember { mutableStateOf(onRequestCompleted, referentialEqualityPolicy()) }
```

Comment:
Ok to drop the `referentialEqualityPolicy()`?

Reply:
Yep. The only thing which reads it isn't composable so there's nothing to recompose.